qiita_db/study.py

```diff
@@ -176,6 +182,41 @@ def get_by_status(cls, status):
         return {x[0] for x in conn_handler.execute_fetchall(sql, (status, ))}
 
     @classmethod
+    def get_info(cls, study_ids, info_cols=None):
+        """Returns study data for a set of study_ids
+
+        Parameters
+        ----------
+        study_cols : list of ints
+            Studies to get information for.
+        info_cols: list of str, optional
+            Information columns to retrieve. Defaults to all study data
+
+        Returns
+        -------
+        list of DictCursor
+            Table-like structure of metadata, one study per row. Can be
+            accessed as a list of dictionaries.
+        """
+        if info_cols is None:
+            info_cols = cls._info_cols
+
+        search_cols = ",".join(sorted(cls._info_cols.intersection(info_cols)))
```

Comment:
Might raise a warning if the user requests information columns that are not in `cls._info_cols`, since right now it silently ignores them, which could lead to bugs if/when there are typos or misspellings